### PR TITLE
Updated packages and configuration files.

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Updates package dependencies, dot files, and other operation related configuration.

# Needed By (Date)
- All of this is in various parts of other PRs so having it all in one place is nice but not urgent. 

# Urgency
- Low

# Steps to Test

1. Check out this branch and delete your node_modules directory.
2. Run `nvm install 11.14.0`
3. Run `nvm use`
4. Run `npm install`
5. Run `npm run build`
6. Run `npm run styleguide`
7. For bonus points run a few more commands.

# Affected Projects or Products
- Just operational things.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by sherakama** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr527-kokt5nn72szdtky8eloghkgr5pof5hu5.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5db3891a53999a8136184392

Visual Diffs:
- https://dashboard.tugboat.qa/5db3891a53999a8136184392/visualdiffs

**Comment by yvonnetangsu** [comment]
Got some errors after running npm run build. I did install node 12.12.0 and is using it and did npm install after that. Would `npm rebuild node-sass` fix it?
Screen Shot 2019-10-25 at 4 54 49 PM

**Comment by sherakama** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr527-kokt5nn72szdtky8eloghkgr5pof5hu5.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5db3891a53999a8136184392

Visual Diffs:
- https://dashboard.tugboat.qa/5db3891a53999a8136184392/visualdiffs

**Comment by sherakama** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr527-kokt5nn72szdtky8eloghkgr5pof5hu5.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5db3891a53999a8136184392

Visual Diffs:
- https://dashboard.tugboat.qa/5db3891a53999a8136184392/visualdiffs

**Comment by sherakama** [comment]
I reverted to node 11 as 12 has issues on osx

**Comment by sherakama** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr527-kokt5nn72szdtky8eloghkgr5pof5hu5.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5db3891a53999a8136184392

Visual Diffs:
- https://dashboard.tugboat.qa/5db3891a53999a8136184392/visualdiffs

**Comment by sherakama** [comment]
Running `npm rebuild node-sass` does fix the problem on my end but I end up having to do this too often. Reverting back to node gets us a few fixes to the audit issues and has fewer issues with OSX.

File: .tugboat/config.yml [hunk 1]
```diff
@@ -1,26 +1,21 @@
...
     commands:
-
-      # Commands that set up the basic preview infrastructure
+      # Used in the base image as setup.
       init:
         - |
           curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.0/install.sh | bash
           NVM_DIR="$HOME/.nvm"
           [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
           nvm install 10.15.1
```
Comment (yvonnetangsu) [inline .tugboat/config.yml]: Do we need to update this to 11.14.0 also?

Comment (sherakama) [reply .tugboat/config.yml]: That would probably be good. Thanks for catching that.

**Comment by sherakama** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr527-kokt5nn72szdtky8eloghkgr5pof5hu5.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5db3891a53999a8136184392

Visual Diffs:
- https://dashboard.tugboat.qa/5db3891a53999a8136184392/visualdiffs

**Comment by yvonnetangsu** [comment]
GTG! Feel free to merge when ready.

**Pull request merged**